### PR TITLE
added msf-goma flavor

### PR DESCRIPTION
**Pull request by kitsao**
References [msf-goma #240](https://github.com/medic/config-msf/issues/240)

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -56,6 +56,8 @@ jobs:
       - fastlane deploy flavor:surveillance_covid19_kenya
       - fastlane deploy flavor:vhw_burundi
     - script:
+        - fastlane deploy flavor:msf_goma
```
Comment (kennsippell) [inline .travis.yml:59]: indenting is off here
you might consider moving it to line 49 to reuse one of the existing jobs

**Review by kennsippell — APPROVED** [review]

File: .travis.yml (hunk 1 above)
Comment (kitsao) [reply .travis.yml]: Addressed.

**Comment by derickl** [comment]
@kitsao is the APK still needed?

**Comment by garethbowen** [comment]
Closing as ancient. Feel free to reopen if needed.

**Pull request closed without merging**